circle.yml

```diff
@@ -19,8 +19,7 @@ dependencies:
 
 test:
   override:
-    - echo "Skipped...Running our uitests on every pull request breaks"
-    - echo "them irregularly and they're badly maintained at the moment."
+    - echo "Beep Boop"
```

Comment:
Well to be honest, I don't know what should be done on circle CI. We aren't technically skipping them now.